styles/proton/catppuccin.user.css

```
@@ -236,6 +236,10 @@
       background-color: var(--navigation-current-item-background-color);
       color: var(--email-item-unread-text-color);
     }
+
+    .message-iframe iframe {
```

Okay, I think this works:

```suggestion
    .message-content:not(.plain) .message-iframe iframe
```